### ISSUE TITLE
amrfinderplus support & conda installation

### ISSUE THREAD

**Issue by JFsanchezherrero** [BigDataBiology/argNorm#10]
Hi there,
Thank you very much for this package, it looks really promising.

I have seen in the documentation here in github that amrfinderplus is supporte but when installing package via pip latest version available to be installed version (0.0.1rc3).

I wonder if you could update the pypi package with the current version of your code.

Also, I wonder if you are planning in generating a conda installation available for this package.

Best regards
Jose

**Comment by luispedro** [comment]
Happy to hear you find it useful.

We will eventually do a conda package, but we feel that this would imply a higher level of polish and trust than we think the package has attained so far.

On the other hand, updating the pyPI package is something we can do, but in the meantime, you can also install directly from github:

```python
pip install https://github.com/BigDataBiology/argNorm/archive/refs/heads/main.zip
```

**Comment by JFsanchezherrero** [comment]
Great! 
Thanks for the tip for installation from the github directory.

Looking forward to Conda/pip package availability!

Best regards

**Comment by luispedro** [comment]
We just released version 0.1.0 on PyPI.

We're probably holding off on doing a conda package for now, though.
